db/schema.rb

```diff
@@ -0,0 +1,63 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_08_21_150440) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.bigint "userId_id", null: false
+    t.bigint "postId_id", null: false
+    t.string "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["postId_id"], name: "index_comments_on_postId_id"
+    t.index ["userId_id"], name: "index_comments_on_userId_id"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.bigint "userId_id", null: false
+    t.bigint "postId_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["postId_id"], name: "index_likes_on_postId_id"
+    t.index ["userId_id"], name: "index_likes_on_userId_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.bigint "authorId_id", null: false
+    t.string "title"
+    t.string "text"
+    t.integer "commentsCounter"
+    t.integer "likesCounter"
```

Comment:
- [ ] From the diagram in the requirements guideline, these should be `author_id`. Kindly rename it.

- [ ] Kindly rename these columns to `comments_counter` and `likes_counter` based on naming conventions